suite/tests/api/ir_x86_2args.h

```diff
@@ -439,3 +439,8 @@ OPCODE(blsr, blsr, blsr, 0, REGARG(EDX), MEMARG(OPSZ_4))
 OPCODE(blsmsk, blsmsk, blsmsk, 0, REGARG(EDX), MEMARG(OPSZ_4))
 OPCODE(blsi, blsi, blsi, 0, REGARG(EDX), MEMARG(OPSZ_4))
 OPCODE(tzcnt, tzcnt, tzcnt, 0, REGARG(EDX), MEMARG(OPSZ_4))
+
+/****************************************************************************/
+/* ADX */
+OPCODE(adox, adox, adox, 0, REGARG(EAX), MEMARG(OPSZ_4))
+OPCODE(adcx, adcx, adcx, 0, REGARG(EAX), MEMARG(OPSZ_4))
```

Comment:
I wonder if you could add a 64-bit version for each.  As an example see the mulx entries where it has a 32-bit and a 64-bit, marking the 64-bit as X64_ONLY:

```
OPCODE(mulx_a, mulx, mulx, X64_ONLY, REGARG(RAX), REGARG(R9), MEMARG(OPSZ_8))
OPCODE(mulx, mulx, mulx, 0, REGARG(EAX), REGARG(EBX), MEMARG(OPSZ_4))
```

(I think the _a was supposed to go with _b -- call it what you like, maybe `adox_x64` and `adox`)